Extensions/SendTo/ClientPickerViewController.swift

```diff
@@ -0,0 +1,191 @@
+
+import UIKit
+import Storage
+import SnapKit
+
+protocol ClientPickerViewControllerDelegate {
+    func clientPickerViewControllerDidCancel(clientPickerViewController: ClientPickerViewController) -> Void
+    func clientPickerViewController(clientPickerViewController: ClientPickerViewController, didPickClients clients: [RemoteClient]) -> Void
+}
+
+struct ClientPickerViewControllerUX {
+    static let TableHeaderRowHeight = CGFloat(50)
+    static let TableHeaderTextFont = UIFont.systemFontOfSize(16)
+    static let TableHeaderTextColor = UIColor.grayColor()
+    static let TableHeaderTextPaddingLeft = CGFloat(20)
+
+    static let DeviceRowTintColor = UIColor(red:0.427, green:0.800, blue:0.102, alpha:1.0)
+    static let DeviceRowHeight = CGFloat(50)
+    static let DeviceRowTextFont = UIFont.systemFontOfSize(16)
+    static let DeviceRowTextPaddingLeft = CGFloat(72)
+    static let DeviceRowTextPaddingRight = CGFloat(50)
+}
+
+/// The ClientPickerViewController displays a list of clients associated with the provided Account.
+/// The user can select a number of devices and hit the Send button.
+/// This viewcontroller does not implement any specific business logic that needs to happen with the selected clients.
+/// That is up to it's delegate, who can listen for cancellation and success events.
+
+class ClientPickerViewController: UITableViewController {
+    var profile: Profile?
+    var clientPickerDelegate: ClientPickerViewControllerDelegate?
+
+    var reloading = true
+    var clients: [RemoteClient] = []
+    var selectedClients = NSMutableSet()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = NSLocalizedString("Send Tab", tableName: "SendTo", comment: "Title of the dialog that allows you to send a tab to a different device")
+        refreshControl = UIRefreshControl()
+        refreshControl?.addTarget(self, action: "refresh", forControlEvents: UIControlEvents.ValueChanged)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: UIBarButtonSystemItem.Cancel, target: self, action: "cancel")
+        tableView.tableFooterView = UIView(frame: CGRectZero)
+    }
+
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        reloadClients()
+    }
+
+    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        if clients.count == 0 {
+            return 1
+        } else {
+            return 2
+        }
+    }
+
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if clients.count == 0 {
+            return 1
+        } else {
+            if section == 0 {
+                return 1
+            } else {
+                return clients.count
+            }
+        }
+    }
+
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let cell = UITableViewCell()
```

Comment:
I've seen this in other places in the app where we instantiate a new table view cell instead of dequeuing one. Is there a reason we're doing this instead of `tableView.dequeueReusableCellWithIdentifier(id, forIndexPath: indexPath)`?